packages/desktop-client/src/components/schedules/DiscoverSchedules.js

```diff
@@ -91,8 +98,6 @@ function DiscoverSchedulesTable({ schedules, loading }) {
       </TableHeader>
       <Table
         rowHeight={ROW_HEIGHT}
-        version="v2"
-        backgroundColor="transparent"
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏this table is missing a `:hover` row highlight.